src/service/config.proto

```diff
@@ -23,7 +24,11 @@ message protocol_gwmp_mapping_v1 {
 
 message protocol_gwmp_v1 { repeated protocol_gwmp_mapping_v1 mapping = 1; }
 
-message protocol_http_roaming_v1 {}
+message protocol_http_roaming_v1 {
+  flow_type flow_type = 1;
+  uint32 dedupe_timeout = 2;
+  bytes path = 3;
```

Comment:
declare what this is? if it's an actual http endpoint/path why is this not a string?

Reply:
I was unaware of "string." I will take a look.

Reply:
I have changed it to "string".